### DISCUSSION TITLE
Attach to an existing process on Windows?

### DISCUSSION THREAD

**Discussion by gerroon** [wezterm/wezterm#4648] [Q&A] [answered]
### What Operating System(s) are you running on?

Windows

### WezTerm version

20231202-083842-f2a9c062

### Ask your question!

Hi
Is there a way to attach an existing running process (far.exe) to new Wezterm tab on Windows? I can do this with ConEmu.

https://conemu.github.io/en/LaunchNewTab.html#Attach_existing_console_or_GUI_application

**Comment by wez** [comment, marked as answer]
No, wezterm doesn't have this function, and cannot support it. ConEmu achieves it by injecting a hook DLL into a target process and then essentially screen-scraping a buffer. That approach has various limitations and downsides and won't be implemented in wezterm.
